datamigration/doc.go

```diff
@@ -0,0 +1,27 @@
+/*
+
+Data migration package implements a functionality to declare and execute data
+migrations.
+
+Each data migration is a function registered under a unique name, that can be
+executed only once.
+
+In order to register a new data migration:
```

Comment:
I found this _steps_ to be insufficient. For example **register a new migration using datamigration.MustRegister** but register in where? in the `cmd/app` folder?

Reply:
I will update this description. The current instruction expects that you have set everything up before and now you only want to add another migration.